util/travis_build.sh

```diff
@@ -22,6 +22,11 @@ for KB in $(make list-keyboards); do
 		continue
 	fi
 
+	if [[ -e keyboards/${KB}/.noci ]]; then
```

Comment:
If here, it will skip building user keymaps, even if that is the only change in the PR. I think there is also still value in `make ${KB}:all` which would also be skipped. Maybe there is a nice way that `.noci` is only looked at when making all?

Reply:
I tried to figure out how to get that done in make and I broke my brain. This was the best compromise I could come up with.